core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -384,6 +390,51 @@ impl SchedulerController {
             );
         }
     }
+
+    /// Calculate priority and cost for a transaction:
+    ///
+    /// Cost is calculated throught the `CostModel`,
+    /// and priority is calculated through a formula here that attempts to sell
+    /// blockspace to the highest bidder.
+    ///
+    /// The priority is calculated as:
+    /// P = R / (1 + C)
+    /// where P is the priority, R is the reward,
+    /// and C is the cost towards block-limits.
+    ///
+    /// Current minimum costs are on the order of several hundred,
+    /// so the denominator is effectively C, and the +1 is simply
+    /// to avoid any division by zero due to a bug - these costs
+    /// are calculated by the cost-model and are not direct
+    /// from user input. They should never be zero.
+    /// Any difference in the prioritization is negligible for
+    /// the current transaction costs.
```

Comment:
nit: hmm, i couldn't understand this sentence well. maybe, this is pointing out the fact `MULTIPLIER` was needed?

Reply:
It's a continuation of the paragraph talking about the effect of +1, since the denominator is (cost + 1) to just avoid any divide by zero.

Since cost is on the order of hundreds to thousands the effect of the +1 in denominator is negligible.